Scalar.Common/Platforms/Mac/MacPlatform.cs

```diff
@@ -148,7 +151,32 @@ public override void IsServiceInstalledAndRunning(string name, out bool installe
 
         public override string GetTemplateHooksDirectory()
         {
-            return Path.Combine("/usr", "local", "git", ScalarConstants.InstalledGit.HookTemplateDir);
+            string gitExecPath = GitInstallation.GetInstalledGitBinPath();
+
+            // Resolve symlinks
+            if (NativeMethods.IsSymlink(gitExecPath))
```

Comment:
According to [the documentation of `readlink()`](https://pubs.opengroup.org/onlinepubs/9699919799/functions/readlink.html), its return value should be `NULL` and `errno` should be set to `EINVAL` if the path does not refer to a symbolic link.

So maybe we can avoid calling `IsSymlink()` altogether (which would also make it unnecessary to jump through hoops to declare the native structs)?

Reply:
I've updated this to use `realpath`.